src/query/expression/tests/it/kernel.rs

```diff
@@ -212,115 +208,3 @@ pub fn test_pass() {
         3,
     );
 }
-
-fn run_filter(file: &mut impl Write, predicate: Column, block: &DataBlock) {
```

Comment:
Why do you delete these codes?

Reply:
`test_data_block_create_with_default_value_and_chunk` need `run_take` function, so i move all `run_*` test util functions to `common.rs`.